### ISSUE TITLE
Custom builds fail with 'ReferenceError: module is not defined'

### ISSUE THREAD

**Issue by sverweij** [lodash/lodash-cli#78]
### steps taken
```
> npm install --global lodash-cli
(...)
> lodash --version
4.0.0
> lodash -o ./lodash.js
```

### expected
lodash-cli generates a lodash.custom.js

### found
An error message (see below)

### notes
- I originally tried with `lodash include=memoize,cloneDeep,flatten,defaults --development --output ./lodash.custom.js` - which gave the same results.
- env:
  - node 5.4.1 (/ npm 3.3.12) on OS X
  - node 4.2.4 (/ npm 2.14.12) on OS X
- `lodash --version` and `lodash --help` work as expected
- lodash-cli 3.10.1 works fine in the same envs

### error message
```
evalmachine.<anonymous>:1
module.exports = require('./lodash.js');
^

ReferenceError: module is not defined
    at evalmachine.<anonymous>:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at /Users/sverweij/.nvm/versions/node/v5.4.1/lib/node_modules/lodash-cli/bin/lodash:3056:10
    at build (/Users/sverweij/.nvm/versions/node/v5.4.1/lib/node_modules/lodash-cli/bin/lodash:3058:6)
    at Object.<anonymous> (/Users/sverweij/.nvm/versions/node/v5.4.1/lib/node_modules/lodash-cli/bin/lodash:3410:3)
    at Module._compile (module.js:397:26)
    at Object.Module._extensions..js (module.js:404:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:429:10)
```

**Comment by jdalton** [comment]
Hi @sverweij!

This should be resolved in the v4.0.1 release (due out in a couple days).

**Comment by arendjr** [comment]
Are you sure about this? I just cloned this repo because I was running into the same issue and wanted to try the fix, but even running the master branch I still had this issue.

Maybe I did something wrong though, because I had to adjust the package.json to still use lodash 4.0.0 or otherwise NPM would complain it could not find 4.0.1-pre. But I'm assuming the fix is in the lodash-cli repo and it should still work even with an unpatched lodash library. Otherwise, what's the best way to install the 4.0.1-pre version to try it out? Thanks!
